spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  context ':format_full_date' do
+    it 'formats a date' do
+      allow_any_instance_of(Date).to receive(:strftime).and_return '45 Septemuary 1978'
+      expect(format_full_date(Date.current)).to eq '45 Septemuary 1978'
```

Comment:
This was by design.  I was testing that a date was formatted, not formatted a certain way, in order to DRY out the format process and create one authoritative source for the way dates are diplayed.  Do you think it's better to create a constant for the format string, test that it is set, and use the constant in the tests?  I pondered this but really liked it this way.  That seems like overkill here, but if the team thinks it's better I'll send it up